store/sqlstore/user_store.go

```diff
@@ -269,29 +306,31 @@ func (us SqlUserStore) UpdateFailedPasswordAttempts(userId string, attempts int)
 
 func (us SqlUserStore) UpdateAuthData(userId string, service string, authData *string, email string, resetMfa bool) (string, error) {
 	updateAt := model.GetMillis()
+	args := []interface{}{updateAt, updateAt, service, authData, userId}
 
 	query := `
 			UPDATE
 			     Users
 			SET
 			     Password = '',
-			     LastPasswordUpdate = :LastPasswordUpdate,
-			     UpdateAt = :UpdateAt,
+			     LastPasswordUpdate = ?,
+			     UpdateAt = ?,
 			     FailedAttempts = 0,
-			     AuthService = :AuthService,
-			     AuthData = :AuthData`
+			     AuthService = ?,
+			     AuthData = ?`
 
 	if email != "" {
-		query += ", Email = lower(:Email)"
+		query += ", Email = lower(?)"
+		args = []interface{}{updateAt, updateAt, service, authData, email, userId}
```

Comment:
What I'd suggest is to just have the initial slice till `authData`, then append email here in the condition. Then append `userId` at the end. Alternatively, we can redo it in squirrel. Let me know what you think.

Reply:
I think squirrel case is good. I'll fix it.